cmd/sys-mgmt-executor/main.go

```diff
@@ -0,0 +1,212 @@
+/*******************************************************************************
+ * Copyright 2019 Dell Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *******************************************************************************/
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+	"time"
+)
+
+// Global variables
+var err error
+
+var usageStr = `
+Usage: ./main service operation		Start app with requested {service} and {operation}
+       -h							Show this message
+`
+
+const (
+	START      = "start"
+	STOP       = "stop"
+	RESTART    = "restart"
+	AppOpenMsg = "This is the docker-compose-executor application!"
+)
+
+// TODO: Externalize these by putting in a properties file.
+const (
+	CoreCommandServiceKey          = "edgex-core-command"
+	CoreDataServiceKey             = "edgex-core-data"
+	CoreMetaDataServiceKey         = "edgex-core-metadata"
+	ExportClientServiceKey         = "edgex-export-client"
+	ExportDistroServiceKey         = "edgex-export-distro"
+	SupportLoggingServiceKey       = "edgex-support-logging"
+	SupportNotificationsServiceKey = "edgex-support-notifications"
+	SupportSchedulerServiceKey     = "edgex-support-scheduler"
+)
+
+// TODO: Externalize these by putting in a properties file.
+var services = map[string]string{
+	SupportNotificationsServiceKey: "Notifications",
+	CoreCommandServiceKey:          "Command",
+	CoreDataServiceKey:             "CoreData",
+	CoreMetaDataServiceKey:         "Metadata",
+	ExportClientServiceKey:         "Export",
+	ExportDistroServiceKey:         "Distro",
+	SupportLoggingServiceKey:       "Logging",
+	SupportSchedulerServiceKey:     "Scheduler",
+}
+
+// usage will print out the flag options for the app.
+func HelpCallback() {
+	msg := fmt.Sprintf(usageStr, os.Args[0])
+	fmt.Printf("%s\n", msg)
+	os.Exit(0)
+}
+
+func main() {
+
+	start := time.Now()
+
+	flag.Usage = HelpCallback
+	flag.Parse()
+
+	fmt.Println(AppOpenMsg)
+
+	// Time it took to start service
+	fmt.Println("Application started in: " + time.Since(start).String())
+
+	var service = ""
+	var operation = ""
+
+	if len(os.Args) > 2 {
+		service = os.Args[1]
+		operation = os.Args[2]
+
+		switch operation {
+		case START:
```

Comment:
You could literally replace this whole switch with 
```
err := ExecuteDockerCommands(service, operation)
if err != nil {
    fmt.Println("error performing ", operation, " on service: ", service)
} else {
    fmt.Println("success performing ", operation, " on service: ", service)
}
```
Not sure why using `fmt.Println` instead of logger.LoggingClient (you have access to it since you're in the mono-repo) but that's another issue.

Reply:
I like that simplification 👍 Also thanks for pointing out those `fmt statements`: They were a carry-over from when this `SMA docker-compose executor` was conceived as located in a standalone repo. Now that it (i.e. `SMA docker-compose executor`) has become a part of edgex-go mono repo, those `fmt statements` have been replace by suitable LoggingClient statements.